lib/Doctrine/ORM/QueryBuilder.php

```diff
@@ -1165,7 +1165,7 @@ public function where($predicates)
      *
      * @return $this
      */
-    public function andWhere()
+    public function andWhere($where)
```

Comment:
This is a breaking change for extending classes, as illustrated here: https://3v4l.org/bIXb5
This is why the situation has been fixed on 3.0.x, by https://github.com/doctrine/orm/commit/77c73e880131de3cd75de3b0c60b8f5276562e14